developer_manual/app_development/bootstrap.rst

```diff
@@ -52,19 +52,24 @@ The class **must** extend ``OCP\AppFramework\App`` and may optionally implement
     namespace OCA\MyApp\AppInfo;
 
     use OCA\MyApp\Listeners\UserDeletedListener;
-    use OCA\MyApp\Notifications\Notifier;
+    use OCA\MyApp\Notification\Notifier;
```

Comment:
Is this referenced somewhere else?

Reply:
``Notification`` is commonly used. If there is no standard, this change may be not necessary.
```php
OCA\Comments\Notification
OCA\ServerInfo\Notification
OCA\Files\Notification
OCA\Files_Sharing\Notification
OCA\FirstRunWizard\Notification
OCA\Support\Notification
OCA\NextcloudAnnouncements\Notification
OCA\UpdateNotification\Notification
OCA\User_LDAP\Notification
```

Exceptions
```php
OCA\Notifications\Notifier
OCA\TwoFactorBackupCodes\Notifications
OCA\DAV\CalDAV\Reminder\
```

Reply:
All of them are other namespaces that are not releated. This one is for the app `MyApp`. All that matters here is (at least from a very limited point of view) that the namespace matches at the import statement and the implementing code. So I would revert it for now.